Sources/Build/BuildDescription/ProductBuildDescription.swift

```diff
@@ -167,6 +167,9 @@ public final class ProductBuildDescription: SPMBuildCore.ProductBuildDescription
         args += ["-L", self.buildParameters.buildPath.pathString]
         args += ["-o", binaryPath.pathString]
         args += ["-module-name", self.product.name.spm_mangledToC99ExtendedIdentifier()]
+        #if swift(>=5.9)
```

Comment:
I think this still isn't quite the check we want because this will happen at compile-time of SwiftPM itself, right? We have `DriverSupport.checkSupportedFrontendFlags()` which could maybe help here.

Reply:
perfect, will do!